cranelift/filetests/filetests/isa/x64/shuffle-avx512.clif

```diff
@@ -16,7 +16,7 @@ block0(v0: i8x16, v1: i8x16):
 ;   movdqa  %xmm0, %xmm5
 ;   movdqu  const(0), %xmm0
 ;   movdqa  %xmm5, %xmm6
-;   vpermi2b %xmm1, %xmm6, %xmm0, %xmm0
+;   vpermi2b %xmm0, %xmm6, %xmm1, %xmm0
```

Comment:
I've verified locally with Intel's emulator that this change still works, so the swapping of `a b` in the above lowering rules should be correct and this is a result of slightly different printing now.